handbook/company/pricing-features-table.yml

```
@@ -705,40 +727,41 @@
     - description: Encrypt hard disks on macOS with FileVault
     - description: Escrow FileVault keys on macOS
     - description: Encrypt hard disks on Windows with BitLocker.
-- industryName: Audit queries and user activities
-  productCategories: [Endpoint operations,Device management]
-  pricingTableCategories: [Endpoint operations]
-  tier: Free
-  usualDepartment: Security
 - industryName: Grant API-only access
+  documentationUrl: https://fleetdm.com/docs/using-fleet/fleetctl-cli#using-fleetctl-with-an-api-only-user
   productCategories: [Endpoint operations]
   pricingTableCategories: [Endpoint operations]
   tier: Free
-- industryName: Programmable audit log
+- industryName: Programmable audit log of system activity (queries, user activities, etc)
+  documentationUrl: https://fleetdm.com/docs/using-fleet/log-destinations
```

```suggestion
  documentationUrl: https://fleetdm.com/docs/rest-api/rest-api#list-activities
```
I realize it was unclear, this feature is about the ability to grab the activity feed in the admin UI.